std/string.d

```diff
@@ -2870,10 +2900,24 @@ if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range) &&
     assert(stripLeft([paraSep] ~ "hello world" ~ paraSep) ==
            "hello world" ~ [paraSep]);
 
+    assert(stripLeft("     hello world     ", " ") ==
```

Comment:
Please put the new unittests in a separate unittest block, and also, please move the new `stripLeft` overload *after* the original unittest block. This is so that the generated documentation will make sense.

In other words, what you currently have is this:
````
/** ... */
auto stripLeft(...) ... // original stripLeft()

/** ... */
auto stripLeft(...) // new stripLeft

///
unittest
{
   ... // original test cases
   ... // new test cases
}
````

What it *should* be is this:

````
/** ... */
auto stripLeft(...) ... // original stripLeft

///
unittest { ... } // original test cases

/** ... */
auto stripLeft(...) ... // new stripLeft

///
unittest { ... } // new test cases
````